### PR TITLE
Rename Rileylinks

### PR DESCRIPTION
**Pull request by ps2**
Need to figure out why peripheralDidUpdateName is not being called.  Likely an issue in the bgscript

### CODE REVIEW

**Comment by loudnate** [comment]
Awesome!

**Comment by loudnate** [comment]
Since you're renaming some enum values anyway, worth changing them to follow the cocoa naming and case conventions? And use the NS_ENUM macro? The Obj-C modernization tool can do the bulk of this for you.

**Comment by ps2** [comment]
This will need the latest updates to the ble113 firmware.

File: RileyLink/RileyLinkBLEDevice.h [hunk 1]
```diff
@@ -11,9 +11,9 @@
 
 
 typedef NS_ENUM(NSUInteger, RileyLinkState) {
-  RILEY_LINK_STATE_CONNECTING,
-  RILEY_LINK_STATE_CONNECTED,
-  RILEY_LINK_STATE_DISCONNECTED
+  RLStateConnecting,
+  RLStateConnected,
+  RLStateDisconnected
 };
```
Comment (loudnate) [inline RileyLink/RileyLinkBLEDevice.h:17]: So the naming conventions state that each type should begin with the enum type name, e.g. `RileyLinkStateConnecting`. This automatically bridges to swift enums as `case Connecting`

**Comment by loudnate** [comment]
LGTM.

File: RileyLink/RileyLinkBLEDevice.h (hunk 1 above)
Comment (ps2) [inline RileyLink/RileyLinkBLEDevice.h:17]: Ah, thanks.

**Comment by ps2** [comment]
Fixes #20

**Pull request merged**